example_iiif_job.json

```diff
@@ -0,0 +1,95 @@
+{
+  "input": {
+    "path": "iiif-media",
+    "depth": 3,
+    "skip": 0,
+    "id": {
+      "name": "SequentialObjectIdGenerator",
+      "properties": {}
+    },
+    "iiif": {
+      "url": "https://libimages.princeton.edu/loris/pudl0001/5138415",
+      "imageApiVersion": "2.1.1",
+      "rotation": 10,
+      "items": [
```

Comment:
If i understand this example correctly, IIIF does not have a discovery mechanism, correct?

Reply:
Image API does'nt have a dicovery mechanism, IIIF has a "Change Discovery" API but it is currently in Beta and no implementing servers are available for us to make use of. Currently, I am working on integrating the Presentation API which let's us download all images in a manifest at once. For example, an entire book can be downloaded using a single URL specified in the job file. You can check out code this at https://github.com/singaltanmay/cineast/tree/feature/IIIF_Presentation_API